wiki/People/Users_with_unique_titles/de.md

```diff
@@ -224,55 +219,12 @@ Die Wettbewerbe [osu!trivium Quiz](/wiki/Contests/osu!trivium_Quiz) vergeben ihr
 - ::{ flag=US }:: [Codazen](https://osu.ppy.sh/users/18291144)
 - ::{ flag=MY }:: [squidstain](https://osu.ppy.sh/users/11073207)
 
-## Featured Artists
-
-[Featured Artists](/wiki/People/Featured_Artists) mit osu!-Accounts bekommen den Titel **Featured Artist**, um ihr Engagement zu zeigen.
+## osu!musician
 
-::{ flag=NO }:: [cYsmix](https://osu.ppy.sh/users/272870) und ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) haben den Titel **osu!musician** für frühere Beiträge zum Sound-Design von osu! und den offiziellen Titelsongs erhalten. ::{ flag=US }:: [BilliumMoto](https://osu.ppy.sh/users/3862471) entschied sich dafür, den Titel **osu!mixer** zu behalten (welcher inzwischen zu osu!mixer II umgewandelt wurde), als er ein Featured Artist wurde.
+Der Titel **osu!musician** wurde an Benutzer für ihre früheren Beiträge zum Sounddesign und zur offiziellen Titelmusik von osu! verliehen.
```

Comment:
```suggestion
Der Titel **osu!musician** wurde an Benutzer für ihre früheren Beiträge zum Sounddesign und den offiziellen Titelsongs von osu! verliehen.
```
I think this actually works better, because it's more clear that it's multiple themes and it stays more true to the english article. 🤔

Reply:
Yes, you're right! I will add it in.